src/app.js

```diff
@@ -28,6 +28,27 @@ app.get('/', function (req, res) {
     routes.root(req, res, db);
 });
 
+app.get('/global-stats.html', function (req, res) {
```

Comment:
I still think these should be just `/global-stats` etc or actually, maybe just `/stats` would be nicer. But definitely without the `.html`. I mean, this is not PHP land ;)